api/routes/locations.js

```diff
@@ -1,21 +1,27 @@
 'use strict';
```

Comment:
Just sticking it up here since I'm not sure where else it should go.

Looking over the new `/locations` output versus the current, the below items come up
- [ ] `coordinates` should be an object, it's showing up as the PostGIS string. Just noticed there is a lat/lon field added, but that can instead be put in the coordinates field.
- [ ] We are missing `sourceName` which we should keep. It should be the last used sourceName, which should hopefully be the active source.
- [ ] In the new output, there is a NULL `name` field, I think that can be removed?
- [ ] For things like `city` and `location` which are now being treated like arrays, I think that will break a lot of things downstream. Can we change those to `cities` and `locations` as arrays, and keep `city` and `location` to be strings, I guess using the last value that was recorded for them?

Reply:
Will fix coordinates and implement changes as proposed.